shell/browser/native_window_mac.mm

```diff
@@ -196,6 +196,9 @@ static bool FromV8(v8::Isolate* isolate,
   params.type = views::Widget::InitParams::TYPE_WINDOW;
   // Allow painting before shown, to be later disabled in ElectronNSWindow.
   params.headless_mode = true;
+  if (transparent()) {
```

Comment:
I'm coming into this with very little context so I could be wrong, but should this be `IsTranslucent()`?

The Views subclass handles it [like this](https://github.com/electron/electron/blob/main/shell/browser/native_window_views.cc#L280):

```
  // If a client frame, we need to draw our own shadows.
  if (IsTranslucent() || has_client_frame())
    params.opacity = InitParams::WindowOpacity::kTranslucent;
```

Reply:
Good catch! Fixed in https://github.com/electron/electron/pull/46353/commits/d24929b2c1b22863bcf94474af2f2ce3af8ed580